rhwp/rhwp-studio/src/ui/agent-sidebar/index.ts

```
@@ -4757,8 +4901,31 @@ export function initAgentSidebar(deps: AgentSidebarDeps): {
           rebuildEffortMenu();
         }
         updateComposer();
+        // Socket-open precedes the authoritative welcome/chat-started frame,
+        // so non-global reference scopes are refreshed only after the hub has
+        // bound this exact thread and document identity.
+        void referenceLibrary.refresh();
         // 새 채팅(welcome)·재시작 시 작업 방식과 계획 단계를 서버와 다시 맞춘다.
         syncPlanningFromBridge();
+        const liveQuestion = bridge.getPendingUserQuestion();
```

Biome (2.5.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)